src/ApiGateways/ApiGw-Base/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM microsoft/dotnet:2.1-aspnetcore-runtime AS base
+FROM microsoft/dotnet:2.1.3-aspnetcore-runtime AS base
```

Comment:
Setting a specific patch version in Dockerfiles would mean that they would have to be updated for every new image release. 

This would probably be a desired production scenario, so versions only change on a schedule and under operations control, but this is not the case in eShopOnContainers, since this is showcase/demo/test app.

Even that "**shouldn't**" be necessary, because we're talking about patch version but then, there are some patches 🙄!

In case the latest stable image (not nightly) causes some problem then the version would be set to the latest known patch that worked fine (as happened at least once), but eventually would be changed back to a fixed minor version.

Anyway let's wait for @CESARDELATORRE's comment on this issue just to confirm.

Reply:
I'd prefer to use the major version of the image if it already supports AspNetCore 2.1.3

Reply:
The minor version you mean, right? (i.e. 2.1)

Reply:
@CESARDELATORRE, could you please confirm the minor version (2.1) should be used here?

Reply:
So, I reverted to 2.1-aspnetcore-runtime in Dockerfiles and checked that it still works for me.